src/Microsoft.AspNet.Mvc.TagHelpers/Internal/JavaScriptResources.cs

```diff
@@ -49,14 +50,11 @@ public static string GetEmbeddedJavaScript(string resourceName)
                 }
             });
         }
-        
+
         private static string PrepareFormatString(string input)
         {
-            // Replace unescaped/escaped chars with their equivalent
-            return input.Replace("{", "{{")
-                        .Replace("}", "}}")
-                        .Replace("[[[", "{")
-                        .Replace("]]]", "}");
+            // Remove final ");". Those characters are in the file only to allow minification.
+            return input.Substring(0, input.Length - 2);
```

Comment:
FYI this class is `static` and has a `ConcurrentDictionary` which will cache this substring.